libs/ledger-live-common/src/families/cosmos/js-synchronisation.ts

```diff
@@ -162,18 +161,11 @@ const txToOps = (info: any, id: string, txs: any): Operation[] => {
 
 export const getAccountShape: GetAccountShape = async (info) => {
   const { address, currency, derivationMode, initialAccount } = info;
-  let xpubOrAddress = address;
-
-  if (address.match("cosmospub")) {
-    const pubkey = decodeBech32Pubkey(address);
-    xpubOrAddress = pubkeyToAddress(pubkey as any, "cosmos");
-  }
```

Comment:
Do you know what this was for? Isn't it useful anymore?...

Reply:
it is not useful anymore